doc/040_backup.rst

```diff
@@ -45,6 +45,17 @@ size of the files and directories in ``~/work`` on the local file system. It
 also tells us that only 1.200 GiB was added to the repository. This means that
 some of the data was duplicate and restic was able to efficiently reduce it.
 
+We just attached the absolute path ``~/work`` to the backup, so the path
```

Comment:
I'd prefer to move this to it's own subsection, for example `Paths in a snapshot` and then show a backup using a relative and an absolute path each followed with the corresponding output of `snapshots` and `ls` as illustration.

The jump from `~/work` to `/home/user/work` may also not be obvious to every user, especially those that are unfamiliar with unix shells.

Reply:
Please do so